README.md

```diff
@@ -206,9 +207,11 @@ the migration. You have to include the following dependency to make it work:
   </dependency>
 ```
 
-In your properties file you will have four new properties that can be set:
+In your properties file you will have new properties that can be set:
 * cassandra.migration.keyspace-name Specifies the keyspace that should be migrated
-* cassandra.migration.script-location Overrides the default script location
+* cassandra.migration.script-locations Overrides the default script location
```

Comment:
You should likely restore the original `cassandra.migration.script-location` and mark it as deprecated so people know they need to migrate. Otherwise this change would break backwards compatibility and as such would require a major release (for no good reason).

Reply:
You're right, thanks for the suggestion